mod/wikirate_source/spec/set/type_plus_right/source/file-import_spec.rb

```diff
@@ -2,191 +2,185 @@
-          expect(cambridge_2015_metric_value_card.content).to eq("800")
+    context 'company correction name is empty' do
+      context 'non-matching case' do
+        it 'should create company and the value' do
+          Card::Env.params[:metric_values] = {"Cambridge"=>['800']}
+
+          source_file = @source.fetch trait: :file
+          source_file.update_attributes subcards: {"#{@source.name}+#{Card[:metric].name}"=>{content: '[[Access to Nutrition Index+Marketing Score]]',type_id: Card::PointerID},"#{@source.name}+#{Card[:year].name}"=>{content: '[[2015]]',type_id: Card::PointerID}}
+          expect(Card.exists?('Cambridge')).to be true
+          expect(Card.exists?('Access to Nutrition Index+Marketing Score+Cambridge+2015')).to be true
+          cambridge_2015_metric_value_card = Card['Access to Nutrition Index+Marketing Score+Cambridge+2015+value']
+          expect(cambridge_2015_metric_value_card.content).to eq('800')
         end
       end
     end
-    context "metric value exists" do
-      it "updates metric values" do
-        Card::Env.params[:metric_values] = {"Amazon.com, Inc."=>["9"]}
-
-        source_file = @source.fetch :trait=>:file
-        source_file.update_attributes :subcards=>{"#{@source.name}+#{Card[:metric].name}"=>{:content=>'[[Access to Nutrition Index+Marketing Score]]',:type_id=>Card::PointerID},"#{@source.name}+#{Card[:year].name}"=>{:content=>'[[2015]]',:type_id=>Card::PointerID}}
+    context 'metric value exists' do
+      it 'updates metric values' do
+        Card::Env.params[:metric_values] = {"Amazon.com, Inc."=>['9']}
 
-        expect(Card.exists?("Access to Nutrition Index+Marketing Score+Amazon.com, Inc.+2015")).to be true
+        source_file = @source.fetch trait: :file
+        source_file.update_attributes subcards: {"#{@source.name}+#{Card[:metric].name}"=>{content: '[[Access to Nutrition Index+Marketing Score]]',type_id: Card::PointerID},"#{@source.name}+#{Card[:year].name}"=>{content: '[[2015]]',type_id: Card::PointerID}}
 
-        amazon_2015_metric_value_card = Card["Access to Nutrition Index+Marketing Score+Amazon.com, Inc.+2015+value"]
-        expect(amazon_2015_metric_value_card.content).to eq("9")
-
-        Card::Env.params[:metric_values] = {"Amazon.com, Inc."=>["999"]}
-        source_file.update_attributes :subcards=>{"#{@source.name}+#{Card[:metric].name}"=>{:content=>'[[Access to Nutrition Index+Marketing Score]]',:type_id=>Card::PointerID},"#{@source.name}+#{Card[:year].name}"=>{:content=>'[[2015]]',:type_id=>Card::PointerID}}
-        expect(amazon_2015_metric_value_card.content).to eq("999")
+        expect(Card.exists?('Access to Nutrition Index+Marketing Score+Amazon.com, Inc.+2015')).to be true
 
+        amazon_2015_metric_value_card = Card['Access to Nutrition Index+Marketing Score+Amazon.com, Inc.+2015+value']
+        expect(amazon_2015_metric_value_card.content).to eq('9')
 
+        Card::Env.params[:metric_values] = {"Amazon.com, Inc."=>['999']}
```

Comment:
Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.